### PR TITLE
38 user verification request

### PR DESCRIPTION
**Pull request by Mateusz-Dobrzynski**
~~This PR is not ready yet, as not all tests pass 🔴~~
The verification request included in this PR will be sent by email to verify an account. This is yet to be implemented.

### CODE REVIEW

File: src/test/java/org/lookout_studios/meals_management_system/meals_management_system/RegistrationServiceTests.java
Comment (hubertborysowski) [inline src/test/java/org/lookout_studios/meals_management_system/meals_management_system/RegistrationServiceTests.java:1]: Line 19: Warning `import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;` is never used.

File: src/main/java/org/lookout_studios/meals_management_system/meals_management_system/VerificationService.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    Logger log = LoggerFactory.getLogger(VerificationService.class);
+
+    @GetMapping(value = "/verify", produces = "application/json")
+    @Tag(name = "user")
+    @Operation(summary = requestSummary, description = requestDescription)
+    @Parameter(name = "registrationToken", required = true, description = registrationTokenDescription)
+    @Parameter(name = "userId", required = true, description = userIdDescription)
+    @ApiResponse(responseCode = "200", description = "${api.response-codes.ok.desc}", content = @Content)
+    @ApiResponse(responseCode = "403", description = "Registration token does not belong to user with provided ID", content = @Content)
+    @ApiResponse(responseCode = "500", description = "${api.response-codes.internalServerError.desc}", content = @Content)
```
Comment (hubertborysowski) [inline src/main/java/org/lookout_studios/meals_management_system/meals_management_system/VerificationService.java]: Shouldn't the description fields be made in the same way as in previous lines - by using Strings?

**Review by hubertborysowski — CHANGES_REQUESTED** [review]
1 Warning found. Potential magic values has been found as well.

**Comment by hubertborysowski** [comment]
By the way, should we consider MySQL queries as magic values when not declared as strings? I probably should've asked that question earlier but it just came to my head now.

**Comment by Mateusz-Dobrzynski** [comment]
Well, this would be the case but the content of the query always depends on the data that is requested. That's why queries are formatted with `String.format()`. While this is not the most elegant solution, it's the best one I could think of. We can always refactor it, should we come up with something better.

File: src/main/java/org/lookout_studios/meals_management_system/meals_management_system/VerificationService.java (hunk 1 above)
Comment (Mateusz-Dobrzynski) [reply src/main/java/org/lookout_studios/meals_management_system/meals_management_system/VerificationService.java]: Here I referenced templates present in `application.properties` to ensure consistency among different requests. Other than that, it's easier to change one property in a single file than to change variables in multiple files, if we decide to change it at some point. In other words, this makes code future-proof.

**Review by hubertborysowski — APPROVED** [review]
Everything works fine, code is understandable, clean and documented.

**Pull request merged**